### DISCUSSION TITLE
What difference between $data and $store?

### DISCUSSION THREAD

**Discussion by bubnenkoff** [alpinejs/alpine#2611] [1. Help]
Hi! Could you explain what difference between $data and $store?
When I need to use one and when another?

**Comment by risangbaskoro** [comment]
AFAIK, `$store` is used when you access global Alpine stores registered using `Alpine.store(...)`.

Meanwhile `$data` is useful when you want to encapsulates all scope to pass around to other script.

It's all in the [`$store`](https://alpinejs.dev/magics/store) and [`$data`](https://alpinejs.dev/magics/store) documentation.
